### PR TITLE
adding focus/hotkey for enter, adjusting width

### PR DESCRIPTION
**Pull request by samageloff**

### CODE REVIEW

**Comment by samageloff** [comment]
[(1) React{ions} - bump width on select field, add hotkey](https://trello.com/c/8a4hYZiA/618-1-react-ions-bump-width-on-select-field-add-hotkey)

File: test/config/index.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
-// require all test .js files
-const testsContext = require.context('../', false, /\.js/)
-//NOTE: in the interest of speed, to test an individual component, use something like this:
-//const testsContext = require.context('../', false, /\RadioGroup.js/);
+require('../SelectField.js')
```
Comment (Chochek) [inline test/config/index.js]: Revert these changes

Comment (samageloff) [inline test/config/index.js]: ⚡️
